docs/3.0.0-beta.x/guides/deployment.md

```diff
@@ -1622,3 +1622,443 @@ heroku open
+
+Create a `Book` type:
+
+```
+model name: book
+title: text
+description: rich text
+```
+
+Save.
+
+Create an `Author` type:
+
+```
+model name: author
+name: text
+```
+
+Save.
+
+Open the `Book` type, add a relation:
+
+```
+book has and belongs to many authors
+```
+
+Save.
+
+Logout, quit the quit running app.
+
+The `api/` folder now includes `book` and `author`.
+
+Deploy.
+
+``` bash
+gcloud app deploy app.yaml --project myapi-123456
+```
+
+It might say `Uploading 0 files`, but if you look at the service sources, you should see the new version.
+
+App Engine > Services > myapi (2 versions) > Source
+
+The contents of the `api` folder were deployed, you can inspect the `book` and `author` folders.
+
+Visit and reload the app URL so that the new version is started up.
+
+```
+https://myapi-123456.appspot.com
+```
+
+Login to the admin UI.
+
+```
+https://myapi-123456.appspot.com/admin
+```
+
+Open Books and Authors, add records and create relations.
+
+If you haven't enabled the permission yet, the API will respond with `403 Forbidden`.
+
+```
+https://myapi-123456.appspot.com/books
+```
+
+Update Permissions:
+
+Roles and Permissions > Public
+
+Enable `count`, `find`, `findone` for `author` and `book`.
+
+```
+https://myapi-123456.appspot.com/books
+```
+
+Should return the JSON data.
+
+### File uploading to Google Cloud Storage
+
+[Lith/strapi-provider-upload-google-cloud-storage](https://github.com/Lith/strapi-provider-upload-google-cloud-storage)
+
+``` bash
+yarn add strapi-provider-upload-google-cloud-storage
+```
+
+Deploy so that the server app includes the dependency from `package.json`.
+
+Create a Google service account key.
+
+<https://console.cloud.google.com/apis/credentials/serviceaccountkey>
+
+Save the JSON credentials file.
+
+Plugins > File Upload > Settings > Production tab
+
+By default `localhost` is selected. Select the `Google Cloud Storage` plugin.
+
+Copy the JSON key and set the regions.
+
+Open the `Cloud Console > Storage > Browser` menu.
+
+Copy the bucket name to the plugin settings, the default is the app ID, such as `myapi-123456.appspot.com`.
+
+(Note that the `Access control` setting of the bucket has to be `Fine-grained`, which is the default.)
+
+Save.
+
+In the Strapi Admin UI:
+
+Roles & Permissions > Public > Upload
+
+Enable `count` and `findone`.
+
+```
+https://myapi-123456.appspot.com/upload/files/count
+```
+
+Should return `{"count":"0"}`.
+
+Files Upload > upload a file.
+
+`/files/count` should now return `1`.
+
+Logout.
+
+Start the local dev server and login to the Admin UI.
+
+``` bash
+yarn develop
+```
+
+Add a Media field to the Book type.
+
+Content Types > Book > Add another field > Media field
+
+```
+name: downloads
+Multiple media
+```
+
+Save.
+
+Deploy, and login to the server Admin UI.
+
+Open one of the books and add downloads. Save.
+
+The `/books` api should display the associated files.
+
+```
+https://myapi-123456.appspot.com/books
+```
+
+### Success!
```

Comment:
Can you remove this section and just add the `ready to go` message in the previous section.
Thank you.